packages/caches/src/tests/weak.ts

```diff
@@ -2,16 +2,22 @@ import * as assert from "assert";
 import { WeakCache } from "../weak.js";
 
 describe("weak least-recently-used cache", function () {
-  it("can hold lots of elements", function () {
+  it("can hold lots of elements", async function () {
+    this.timeout(5000);
     const cache = new WeakCache();
     const count = 1000000;
     const keys = [];
 
+    console.time("filling up cache");
```

Comment:
Comments are in for local testing right now, we can remove those after you had a chance to play with it.